### PR TITLE
Add visual diff back

### PR DESCRIPTION
**Pull request by svanherk**
Use the new visual diff action!  Putting this in its own workflow so it can be rerun without rerunning everything else in CI.  I've turned off most tests for now, since we're going to see A LOT of failures.  Will open up PRs adding them back in groups, to make things easier to review.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -7,7 +7,5 @@ updates:
     ignore:
       - dependency-name: "@formatjs/intl-pluralrules"
       - dependency-name: "intl-messageformat"
-      - dependency-name: "karma-browserstack-launcher"
-      - dependency-name: "karma-sauce-launcher"
```
Comment (dlockhart) [inline .github/dependabot.yml:11]: Nice, thank-you.

File: README.md [hunk 2]
```diff
@@ -111,19 +111,25 @@ npm run test:headless:watch
...
 
+The golden snapshots in source control must be updated by Github Actions.  If your PR's code changes result in visual differences, a PR with the new goldens will be automatically opened for you against your branch.
+
+If you'd like to run the tests locally to help troubleshoot or develop new tests, you can use these commands:
+
 ```shell
+# Install dependencies locally
+npm i mocha -g
+npm i @brightspace-ui/visual-diff puppeteer --no-save
+
```
Comment (dlockhart) [inline README.md:122]: I'll take a look at this whole README snippet when we port this over to the `npm init` repo.

File: package-lock.json [hunk 3]
```diff
@@ -8784,26 +8510,6 @@
       "integrity": "sha512-XRsRjdf+j5ml+y/6GKHPZbrF/8p2Yga0JPtdqTIY2Xe5ohJPD9saDJJLPvp9+NSBprVvevdXZybnj2cv8OEd0A==",
       "dev": true
     },
-    "puppeteer": {
```
Comment (dlockhart) [inline package-lock.json:8787]: This alone saves like a minute each `npm install` 🎉

File: package.json [hunk 4]
```diff
@@ -18,9 +18,6 @@
     "start": "es-dev-server --node-resolve --dedupe --watch --open",
     "test": "npm run lint && npm run test:headless && npm run test:axe && npm run test:diff",
```
Comment (dlockhart) [inline package.json:19]:
```suggestion
    "test": "npm run lint && npm run test:headless && npm run test:axe",
```

The README also says this which can be changed:
> # lint, unit test and visual-diff test

**Review by dbatiste — APPROVED** [review]

**Review by dbatiste — APPROVED** [review]

**Review by dlockhart — APPROVED** [review]

**Pull request merged**